cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1067,4 +1067,8 @@ public static boolean deleteClaim(StatefulSet ss) {
     String getAncillaryConfigMapKeyLogConfig() {
         return ANCILLARY_CM_KEY_LOG_CONFIG;
     }
+
+    public static String getClusterCaName(String assemblyName)  {
```

Comment:
we are calling `cluster` this kind of parameter for all the similar methods in the subclasses. From the PR description it should be the same, i.e. my-cluster-cluster-ca